package.json

```diff
@@ -68,5 +70,17 @@
       "pre-commit": "lint-staged"
     },
     "homepage": "https://github.com/andela/king-kong-ah-backend#readme"
+  },
+  "nyc": {
+    "all": true,
+    "include": [
+      "server/**/*.js"
+    ],
+    "exclude": [
+      "**/*.spec.js",
+      "server/migrations",
+      "server/models/index.js",
+      "server/app.js"
```

Comment:
it's because of the `all:true` you added that is making it test for this file `app.js`, `server/migrations` which it was supposed to be  ignored by default
Now you are now telling it to ignore it again

You can do something like this

```
"nyc": {
    "exclude": [
      "server/models/index.js",
      "server/tests"
    ]
  }
```